tests/unit/testUncaught.js

```diff
@@ -50,6 +52,13 @@ test('Uncaught handler setup', function (t) {
 });
 
 test('Test uncaught shutdown behavior', function (t) {
+  if (!isString(process.env.GOOGLE_APPLICATION_CREDENTIALS)
```

Comment:
Ah, that is a good point. Why is `process.env.GOOGLE_APPLICATION_CREDENTIALS` required in this test again? If we aren't actually publishing maybe it and `GCLOUD_PROJECT` could be set to dummy values for testing only as is done in the trace agent?